src/main/java/com/ss/mqtt/broker/model/SharedSubscriber.java

```diff
@@ -0,0 +1,50 @@
+package com.ss.mqtt.broker.model;
+
+import com.ss.mqtt.broker.model.topic.SharedTopicFilter;
+import com.ss.mqtt.broker.network.client.MqttClient;
+import com.ss.rlib.common.util.array.Array;
+import com.ss.rlib.common.util.array.ConcurrentArray;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collection;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class SharedSubscriber implements Subscriber {
+
+    private final SharedTopicFilter topicFilter;
```

Comment:
missed annotations

Reply:
Is it really required ?
Method com.ss.mqtt.broker.model.SubscribeTopicFilter#getTopicFilter returns NotNull value.